src/pudl/metadata/classes.py

```diff
@@ -576,7 +576,7 @@ class Field(Base):
 
     name: SnakeCase
     type: Literal[  # noqa: A003
-        "string", "number", "integer", "boolean", "date", "datetime", "year"
+        "string", "number", "integer", "boolean", "date", "datetime", "year", "category"
```

Comment:
`category` is not an allowed field type under the TableSchema definition, so this violates the referenced standard.

https://specs.frictionlessdata.io/table-schema/#types-and-formats

Under the TableSchema specification, this functionality is implemented with a type (e.g. `string`) plus an `enum` constraint that enumerates the values the field is allowed to take on, so we should probably figure out how to implement the desired functionality within that framework.

Reply:
Got it. I'll take out all the `category` types and instead add an `enum` constraint. These fields can be cast to categoricals later down the line, with `Field.to_pandas_dtype`

Reply:
Yeah, but somehow we have to convey to `to_pandas_dtype()` that it should be treated as a categorical, not just a vanilla string.  I think that that's what happens if you've got an enum constrained string, so if you add the enum in the PPE overrides, it should work maybe?